diesel_cli/src/main.rs

```diff
@@ -606,6 +607,31 @@ fn regenerate_schema_if_file_specified(
     Ok(())
 }
 
+// List which features have been compiled into this version of diesel_cli
+#[cfg(any(feature = "postgres", feature = "mysql", feature = "sqlite"))]
+fn show_which_engines_supported() -> Result<(), Box<dyn Error + Send + Sync + 'static>> {
+    let mut features: Vec<&str> = Vec::new();
+    #[cfg(feature = "postgres")]
+    features.push("postgres");
+    #[cfg(feature = "mysql")]
+    features.push("mysql");
+    #[cfg(feature = "sqlite")]
+    features.push("sqlite");
+
+    for f in &features {
+        match f as &str {
+            "postgres" | "sqlite" | "mysql" | "sqlite-bundled" => print!("{} ", f),
```

Comment:
Where is `sqlite-bundled` coming from?

Reply:
https://github.com/diesel-rs/diesel/blob/master/diesel_cli/README.md#installation
Line 33 `cargo install diesel_cli --no-default-features --features "sqlite-bundled"`

Reply:
Yeah, but it's meaningless here, isn't it? Since you are not pushing it into `features`.